qa/L2_RN50_data_perf/test.sh

```diff
@@ -15,6 +15,8 @@ test_body() {
     # fp16 NHWC
     python test_RN50_data_pipeline.py --gpus ${NUM_GPUS} -b 256 --workers 3 --prefetch 2 --fp16 --nhwc
     python test_RN50_data_pipeline.py --gpus ${NUM_GPUS} -b 16 --workers 3 --prefetch 11 --fp16 --nhwc
+    # FW iterators
+    python test_RN50_data_fw_iterators.py --gpus ${NUM_GPUS} -b 256 --workers 3 --prefetch 2
```

Comment:
I think, we can have `--epochs 3` to test how the iterators "reset"

Reply:
done